src/feedback/trulens/feedback/v2/feedback.py

```diff
@@ -475,6 +509,12 @@ class PromptResponseRelevance(Relevance, WithPrompt, CriteriaOutputSpaceMixin):
         """
     )
 
+    system_prompt: ClassVar[str] = cleandoc(
```

Comment:
I guess the same question as above - we are already doing in `def relevance(...)` in llm_provider.py`   
```
system_prompt = (
            feedback_v2.PromptResponseRelevance.generate_system_prompt(
                min_score_val, max_score_val, criteria, output_space
            )
        )
```, do this not work anymore?

Reply:
This was not working afaict. I ran some test prints of the prompts and the template was not filled in properly, for example the output space was left as "LIKERT_0_3" instead of the prompt itself and the min/max scores were not filled into the criteria.

Reply:
hmm this is odd. 

The code has been there for several months and RAG Triad has been using this logic without issue, at least with the default output space and criteria.


I tested below with a minimal example:
```
from trulens.providers.openai import OpenAI

gpt_4o_mini = OpenAI(model_engine="gpt-4o-mini")

gpt_4o_mini.context_relevance_with_cot_reasons(question='test question', 
                              context='test context',
                              criteria="Daniel's test criteria",
                              min_score_val=0,
                              max_score_val=1)
```

could you share your code that fails? happy to talk synchronously over a call / huddle too. 
and the printed system prompt also shows the logic works properly (I inserted a print within `context_relevance_with_cot_reasons` method) 
```
system_prompt in context relevance w/ cot: You are a RELEVANCE grader; providing the relevance of the given CONTEXT to the given QUESTION.
Respond only as a number from 0 or 1, where 0 is lowest and negative (i.e. irrelevant or not grounded) and 1 is highest and positive (relevant, grounded, valid, etc.).

Criteria for evaluating relevance:
Daniel's test criteria

A few additional scoring guidelines:

- Long CONTEXTS should score equally well as short CONTEXTS.

- RELEVANCE score should increase as the CONTEXTS provides more RELEVANT context to the QUESTION.

- RELEVANCE score should increase as the CONTEXTS provides RELEVANT context to more parts of the QUESTION.

- Never elaborate.
```